### ISSUE TITLE
De-uglify tables in docs

### ISSUE THREAD

**Issue by SouthEndMusic** [Deltares/Ribasim#1723]
I noticed that the tables in the docs now look like this:

![image](https://github.com/user-attachments/assets/2a394e9d-2fa8-4f82-a045-222a77534057)

**Comment by Jingru923** [comment]
> Might be a quarto issue then?

I think so as well

**Comment by visr** [comment]
Yes, what's strange is that https://deltares.github.io/Ribasim/reference/node/basin.html#static is squashed but https://deltares.github.io/Ribasim/reference/node/pump.html#static is not, whereas the tables are almost the same. Both don't have horizontal lines, I think they used to all have those? If I copy the .qmd from both pages together into `table.qmd` and run `quarto render table.qmd` both look good:

![image](https://github.com/user-attachments/assets/8b29a018-3abd-4463-966a-95596d028b59)

Perhaps this is only triggered if it is part of a quarto website project.

**Comment by visr** [comment]
This seems to be fixed, likely by a Quarto update.
